### ISSUE TITLE
Replicating results

### ISSUE THREAD

**Issue by ilia10000** [SsnL/dataset-distillation#10]
I think your paper is fascinating so I have been experimenting with it for a few weeks now.
I was wondering what hyperparams you used to get 10 images that achieve almost 94% accuracy on MNIST after 1 GD step and 3 epochs. I can't seem to hit this when I run the suggested code for 200 epochs. At most I managed to get around 91%. 

`python3 main.py --mode distill_basic --dataset MNIST --arch LeNet --distill_steps 1 --train_nets_type known_init --n_nets 1 --test_nets_type same_as_train`

**Comment by SsnL** [comment]
Thanks for your interest in our work!  It turned out indeed that I set the default training epochs to be too low for some of the settings. In https://github.com/SsnL/dataset-distillation/commit/3d07dd5d71804ffcf89d2618cc6280e1755e56d9 I changed the default values to ones that should cover all settings. I also specified the initial `distill_lr` for some experiments, although in my experience it doesn't matter much.

Thanks again. Let me know if you still have problems using the updated settings.

**Comment by ilia10000** [comment]
Thanks, already getting better results with the new defaults!
Is the reported result in the paper based on the final distilled images (ie images after epoch 350) or is it based on the best intermediate result (e.g. one of the checkpoints at earlier epoch)?

**Comment by SsnL** [comment]
@ilia10000 The results in the paper are based on the final images. It is possible that some intermediate results are better.
